systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -48,14 +48,35 @@
 
     private StUtils() { }
 
-    public static void waitForReconciliation(String testClass, String testName, String namespace) {
-        LOGGER.info("Waiting for reconciliation");
-        String reconciliation = timeMeasuringSystem.startOperation(Operation.NEXT_RECONCILIATION);
-        TestUtils.waitFor("Wait till another rolling update starts", Constants.CO_OPERATION_TIMEOUT_POLL, Constants.RECONCILIATION_INTERVAL + 30000,
-            () -> !cmdKubeClient().searchInLog("deploy", "strimzi-cluster-operator",
-                timeMeasuringSystem.getCurrentDuration(testClass, testName, reconciliation),
-                "'Triggering periodic reconciliation for namespace " + namespace + "'").isEmpty());
-        timeMeasuringSystem.stopOperation(reconciliation);
+    /**
+     * Method waitForPodsStability ensuring for every pod listed for kafka or zookeeper statefulSet will be controlling
+     * their status in Running phase. If the pod will be running for selected time #Constants.GLOBAL_RECONCILIATION_COUNT
+     * pod is considered as a stable. Otherwise this procedure will be repeat.
+     * @param pods all pods that will be verified
+     */
+    public static void waitUntilPodsStability(List<Pod> pods) {
```

Comment:
Just thinking...shouldn't be this in `PodUtils` ?

Reply:
fair point.. done )